README.md

```diff
@@ -24,6 +22,69 @@ Or install it yourself as:
 gem install modulr-api
 ```
 
+## Usage
+
+Run `bin/console` for an interactive prompt to experiment with the code.
+
+### Customers
+
+```rb
+# Find a customer
+client.customers.find(id: "C2188C26")
+```
+
+### Accounts
+
+```rb
+# Find an account
+client.accounts.find(id: "A2188C26")
+
+# Create an account
+client.accounts.create(customer_id: "C2188C26", currency: "EUR", product_code: "YOUR_PRODUCT_CODE", external_reference: "My new account")
+```
+
+### Payments
+
+```rb
+# Find a payment
+client.payments.find(id: "P210FFT5AW")
+
+# List payments
+client.payments.list(from: DateTime.now - 1, to: DateTime.now)
+
+# Create a payment
+client.payments.create(account_id: "A2188C26", currency: "EUR", amount: 0.01, destination: { type: "IBAN", iban: "ES8601280011390100072676", name: "Aitor García Rey" }, reference: "The reference")
+```
+
+### Notifications
+
+### Supported event types per channel
+
+Not all notifications can be sent to any channel. Check the following list for a quick view and the [original](https://modulr.readme.io/docs/notifications-1) reference for an up-to-date list
+
+Supported via webhook:
+
+- ACCOUNT_SWITCH_UPDATE
```

Comment:
There are some events I cannot find in modulr doc, @aitor could you provide a link to the doc which contains all these events?
Example of missing events:
```
ACCOUNT_SWITCH_UPDATE, UPCOMING_COLLECTION_CREDIT(aka UPCOMINGCOLLECTIONCREDIT), UPCOMING_COLLECTION_DEBIT (aka UPCOMINGCOLLECTIONDEBIT) ...
```

Reply:
They can be seen in the docs, e.g. https://modulr.readme.io/reference/addpartnernotification